graphene/types/inputobjecttype.py

```diff
@@ -23,8 +23,8 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         dict.__init__(self, *args, **kwargs)
-        for key, value in self.items():
-            setattr(self, key, value)
+        for key in self._meta.fields.keys():
+            setattr(self, key, self.get(key, None))
```

Comment:
Looks good to me, doesn't really make sense to return the field unless it is later used to validate data generated or retrieve from somewhere else other than user input.

Reply:
Right...and returning a combination of fields and values is a bit confusing.